lib/train/file/remote/aix.rb

```diff
@@ -12,6 +12,12 @@ def link_path
             @backend.run_command("perl -e 'print readlink shift' #{@spath}").stdout.chomp
         end
 
+        def shallow_link_path
```

Comment:
Should we just alias this if its going to be exactly the same as `link_path`?

Reply:
I don't really want to do that. Turns out on aix, `link_path` doesn't respect the spec (which is doing a recursive lookup), this is a problem that should be fixed independently